documentation/book/con-custom-resources-example.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-example-{context}']
+= {Productname} custom resource example
+Each {Productname}-specific custom resource is defined within the specification of a CRD.
+
+To understand the relationship between a CRD and a custom resource, let's look at a sample of the CRD for a Kafka topic.
+
+.Kafka topic CRD
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: CustomResourceDefinition
+Metadata: <1>
+  name: kafkatopics.kafka.strimzi.io
+  labels:
+    app: strimzi
+Spec: <2>
+  group: kafka.strimzi.io
+  versions:
+    {ApiVersion}
+  scope: Namespaced
+  names:
+    # ...
+    plural: kafkatopics
+    shortNames:
+    - kt <3>
+  additionalPrinterColumns: <4>
+      # ...
+  Validation: <5>
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          properties:
+            partitions:
+              type: integer
+              minimum: 1
+            replicas:
+              type: integer
+              minimum: 1
+              maximum: 32767
+      # ...
+----
+<1> The metadata for the topic custom resource, its name and a label to identify the resource.
+<2> The specification for the topic, including the group (domain) name, the plural name and the supported schema version, which are used in the URL to access the API of the topic. The other names are used to identify the resource in the CLI.
```

Comment:
* This is the specification of the CRD, not a topic: "The specification for the topic" → "The specification for this CRD"
* "The other names are used to identify the resource in the CLI." → "The other names are used to identify instance resources in the CLI, for example `kubectl get kafkatopic my-topic` or `oc get kafkatopics`"

Reply:
Updated callout to:

<2> The specification for this CRD, including the group (domain) name, the plural name and the supported schema version, which are used in the URL to access the API of the topic. The other names are used to identify instance resources in the CLI. For example, `kubectl get kafkatopic my-topic` or `oc get kafkatopics`.